src/main/groovy/com/ofg/uptodate/finder/MavenLatestDependenciesProvider.groovy

```diff
@@ -0,0 +1,61 @@
+package com.ofg.uptodate.finder
+
+import com.ofg.uptodate.LoggerProxy
+import groovy.transform.PackageScope
+import groovy.util.logging.Slf4j
+import groovyx.net.http.HTTPBuilder
+
+import java.util.concurrent.Future
+
+import static com.ofg.uptodate.UrlEspaceUtils.escape
+
+@Slf4j
+@PackageScope
+class MavenLatestDependenciesProvider implements LatestDependenciesProvider {
+    
+    private final LoggerProxy loggerProxy
+
+    MavenLatestDependenciesProvider(LoggerProxy loggerProxy) {
+        this.loggerProxy = loggerProxy
+    }
+    
+    @Override
+    List<Dependency> findLatest(List<Dependency> dependencies, FinderConfiguration finderConfiguration) {
+        int httpPoolSize = Math.min(dependencies.size(), finderConfiguration.httpConnectionSettings.poolSize)
+        HTTPBuilder httpBuilder = new HTTPBuilderProvider(finderConfiguration.httpConnectionSettings).withPoolSize(httpPoolSize).get()
+        Closure latestFromMavenGetter = getLatestFromMavenCentralRepo.curry(httpBuilder, finderConfiguration.excludedVersionPatterns)
+        return dependencies.collect(latestFromMavenGetter).collect{it.get()}.grep(getOnlyNewer).collect {it[1]}
+    }
+
+    private Closure<Future> getLatestFromMavenCentralRepo = {HTTPBuilder httpBuilder, List<String> versionToExcludePatterns, Dependency dependency ->
```

Comment:
Same logic as in JCenterLatestDependenciesProvider but only one of those logs.
Do we have any other inconsistencies/duplication between maven and JCenter providers?